src/App.jsx

```diff
@@ -1,11 +1,29 @@
 import { useState } from 'react'
+import ky from 'ky'
 import { Loader } from './components/Loader.jsx'
 import { ErrorMessage } from './components/ErrorMessage.jsx'
 import { TornReviveData } from './components/TornReviveData.jsx'
 import './App.scss'
 
 function App() {
   const [state, setState] = useState('idle');
+  const [userApiValue, setUserApiValue] = useState([])
+
+  const apiHandler = async (userApiValue) => {
+    const search = userApiValue
+    try {
+        setState('loading')
+        const response = await ky.get(`https://api.torn.com/user/?selections=revives&key=${search}`).json()
+        console.log(response)
```

Comment:
NIT: It's usually bad practice to keep `console.log` in a codebase outside of debugging purposes.

Unless you have a strong reason to keep this I would recommend removing it.

Reply:
Gone.